src/main/java/stanhebben/zenscript/util/MethodOutput.java

```diff
@@ -1046,10 +1070,15 @@ public void aThrow() {
         visitor.visitInsn(ATHROW);
     }
     
-    public void position(ZenPosition position) {
+    public Label position(ZenPosition position) {
         Label label = new Label();
+        if(firstLabel == null) {
+            firstLabel = label;
+        }
+        lastLabel = label;
```

Comment:
Since this now became a bit more duplicated code you could delegate to label?
``` java
posision(pos) {
    Label label = new Label
    this.label(label);
    visitor.visitLineNumber(...);
}
```

Reply:
Yes, the first label would be initialized later when the first expression compiles.